bin/make_ctss.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
```

Comment:
As usual with scientific software, the copyright is a bit unclear, since this is the results of several PhD students in my lab polishing the code over the years (mostly without git 😱 , so we can't really track/blame, who actually touched what). So I'll leave it for now how it is...

Reply:
Haha, ok fair enough!